board/recalbox/fsoverlay/recalbox/scripts/recalbox-includes.sh

```diff
@@ -68,3 +75,8 @@ function doRbxConfUpgrade {
   cp "$rbxVersion" "$curVersion" || { recallog -e "ERROR : Couldn't set the new recalbox.conf version" ; return 1 ; }
   recallog "UPDATE done !"
 }
+
+
+function updateBoot {
+  sed -i '/^dtparam=/{h;s/=.*/=audio=on/};${x;/^$/{s//dtparam=audio=on/;H};x}' /boot/config.txt
```

Comment:
could be worth adding some recallog + return value, should we ever fail an update